doc/classes/RichTextLabel.xml

```diff
@@ -24,10 +24,17 @@
 			<param index="3" name="color" type="Color" default="Color(1, 1, 1, 1)" />
 			<param index="4" name="inline_align" type="int" enum="InlineAlignment" default="5" />
 			<param index="5" name="region" type="Rect2" default="Rect2(0, 0, 0, 0)" />
+			<param index="6" name="key" type="Variant" default="null" />
+			<param index="7" name="pad" type="bool" default="false" />
+			<param index="8" name="tooltip" type="String" default="&quot;&quot;" />
+			<param index="9" name="size_in_percent" type="bool" default="false" />
```

Comment:
The full list of parameters is starting to be quite convoluted, that will be an awkward API to use IMO.
Struct support would be great here, passing a configuration struct with selected optional members defined would probably be better.